projects/core/src/user/facade/user-consent.service.spec.ts

```diff
@@ -232,6 +240,9 @@ describe('UserConsentService', () => {
       describe('when the user is anonymous', () => {
         it('should not call getConsents()', () => {
           spyOn(authService, 'isUserLoggedIn').and.returnValue(of(false));
+          spyOn(userIdService, 'getUserId').and.returnValue(
+            of(OCC_USER_ID_ANONYMOUS)
+          );
```

Comment:
It would be good to also add an unit test against the bug, to avoid a possible regression in the future.